platform-configs/minnowboard-turbot.robot

```diff
@@ -47,5 +93,6 @@ Power On
     Rte Power Off
     Sleep    1s
     Telnet.Read
+    Rte Relay Set    on
     Rte Power On
```

Comment:
Relay will already cause the platform to stat booting. The `Rte Power On` is obsolete

Reply:
Fixed in [this commit](https://github.com/Dasharo/open-source-firmware-validation/pull/441/commits/0b52c12d8825278cea0abf2e2b9dc6aa2a5ca401)